content/blog/2017/2017-02-07-gsoc2017-announcement.adoc

```diff
@@ -0,0 +1,72 @@
+---
+:layout: post
+:title: "Jenkins Google Summer Of Code 2017. Call for mentors"
+:tags:
+- gsoc
+- events
+- general
+- gsoc2017
+:author: oleg-nenashev
+---
+
+[NOTE]
+====
+This is a guest post by
+link:https://github.com/oleg-nenashev[Oleg Nenashev],
+one of the Jenkins Organization Admins in Google Summer of Code.
+====
+
+On behalf of the GSoC org team I am happy to announce that we are going to apply to 
+link:https://developers.google.com/open-source/gsoc/[Google Summer of Code (GSoC) on this year]. 
+GSoC presumes that high-profile students work in open-source projects for several months under mentorship of organization members.
```

Comment:
I'm not sure what verb goes here instead of "presumes that", but there should be a different verb.  

"lets"?  "allows"?